params/protocol_params.go

```diff
@@ -150,13 +150,16 @@ type CodeFormat uint8
 
 const (
 	CodeFormatEVM CodeFormat = iota
+	CodeFormatEVM2
```

Comment:
We need more information to distinguish `CodeFormatEVM` and `CodeFormatEVM2`. How about adding comments or changing its name like  `CodeFormatEVMIstanbulCompatible`.

Reply:
I changed the name of CodeFormatEVM2 and added some description. Reflected at https://github.com/klaytn/klaytn/pull/912/commits/1c0928ba549158af0d357eed0e6d0a7763e14bd8.